packages/lesswrong/components/posts/PostsPage/PostActions.tsx

```diff
@@ -172,6 +173,11 @@ const PostActions = ({post, closeMenu, classes}: {
 
   const isRead = (post._id in postsRead) ? postsRead[post._id] : post.isRead;
 
+  const defaultLabel = forumSelect({
```

Comment:
I think "defaultLabel" is the wrong word for the variable which is "either the default label, or a different forum-specific label". I'd maybe say "frontpageMessage" or something.

(Also the fact that the same code is duplicated triggers my "should this just be abstracted?" sensors, although I also do think there's risk from having different UI elements share code, which can result in future devs being surprised when changing something in one place changes it in another place that might evolve UI-wise in different directions)